01 Operator_Overloading/01 OperatorOverloading/OperatorOverloading.DBL/FileExchangeRateProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Parse;
+using System.IO;
+using System.Configuration;
+
+namespace OperatorOverloading.DBL
+{
+    public class FileExchangeRateProvider:IExchangeRateProvider
+    {
+       
+     //   public string Path = "C:/Users/arsalang/Desktop/json.txt";
+        public string Path = ConfigurationManager.AppSettings["Path"];
+        public static Dictionary<string, double> Dictionary = new Dictionary<string, double>();
```

Comment:
why is dictionary public? Name of this parameter should not be dictionary, it be something like _exchangeRates.